# HG changeset patch
# User Rik <rik@octave.org>
# Date 1373573061 25200
#      Thu Jul 11 13:04:21 2013 -0700
# Node ID ec6ced0ece879813c76c183919fc583ade799b1e
# Parent  9f50dd7ece60b8e86650dd74ca08680ffd02cdac
trace.m: Remove accidental documentation test code from 9c971fa62a77

* scripts/linear-algebra/trace.m: Remove @deftypefn test code.

diff --git a/scripts/linear-algebra/trace.m b/scripts/linear-algebra/trace.m
--- a/scripts/linear-algebra/trace.m
+++ b/scripts/linear-algebra/trace.m
@@ -12,17 +12,16 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Hello World} {} trace (@var{A})
 ## @deftypefnx {Function File} {} trace (@var{A})
 ## Compute the trace of @var{A}, the sum of the elements along the main
 ## diagonal.
 ##
 ## The implementation is straightforward: @code{sum (diag (@var{A}))}.
 ## @seealso{eig}
 ## @end deftypefn
 
